nrf_rpc/doc/img/EDITING.TXT

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Why is this file here ?
Do we expect our users to edit those images.

Reply:
Those SVG files are not easy to edit and it is not obvious that `diagrams.net` has to be used for them. There are two reasons that I think that this file should be kept:
1. I know how to edit them, colleagues from my team too, but after few months or years this knowledge may be forgotten.
2. We are publishing open source software. We are allowing to use and modify it. Why should we make those file harder to access for others? Is it not against the open-source model?

Reply:
@tejlmand Is my response satisfying? If not, what you suggesting to do and why?

Reply:
yes, btw. is it on purpose the file name is in pure upper case ? `EDITING.TXT`